### PR TITLE
fix missing serialization tag

### PR DESCRIPTION
**Pull request by deads2k**
@liggitt  missing json tag and some godoc.

### CODE REVIEW

**Comment by deads2k** [comment]
@liggitt Serialization was fixed by #1440.  Might still be worth the godoc, but I don't feel strongly.  Let me know.

File: pkg/cmd/server/api/types.go [hunk 1]
```diff
@@ -154,17 +158,24 @@ type EtcdConfig struct {
...
+	// ServicesSubnet is the subnet to use for assigning service IPs
+	ServicesSubnet string
+	// StaticNodeNames is the list of nodes that are statically known
+	StaticNodeNames []string
+	// SchedulerConfigFile points to a file that describes how to set up the scheduler.  If empty, you get the default scheduling rules.
 	SchedulerConfigFile string
 }
 
 type CertInfo struct {
+	// CertFile is a file holding the public half of public/private keypair
```
Comment (liggitt) [inline pkg/cmd/server/api/types.go]: don't mention keypairs... I don't think that's correct. "CertFile is a file containing a PEM-encoded certificate"

File: pkg/cmd/server/api/types.go [hunk 2]
```diff
@@ -154,17 +158,24 @@ type EtcdConfig struct {
...
 	CertFile string
-	KeyFile  string
+	// KeyFile is a file holding the private half of public/private keypair
```
Comment (liggitt) [inline pkg/cmd/server/api/types.go]: don't mention keypairs... I don't think that's correct. "KeyFile is a file containing a PEM-encoded private key for the certificate specified by CertFile"

File: pkg/cmd/server/api/types.go [hunk 3]
```diff
@@ -154,17 +158,24 @@ type EtcdConfig struct {
...
-	StorageDir    string
+	// StorageDir indicates where to save the etcd data
+	StorageDir string
 }
 
 type KubernetesMasterConfig struct {
-	MasterIP            string
-	ServicesSubnet      string
-	StaticNodeNames     []string
+	// MasterIP is the public IP address of kubernetes stuff.  If emtpy, the first result from net.InterfaceAddrs will be used.
```
Comment (liggitt) [inline pkg/cmd/server/api/types.go]: s/emtpy/empty/

**Comment by liggitt** [comment]
LGTM otherwise

**Comment by deads2k** [comment]
[merge]

**Pull request merged**